update(docs): revamped and expanded documentation (#12)
cross-seed/cross-seed.org: docs/reference/api.md

# HTTP API

`cross-seed` has an HTTP API as part of [Daemon Mode](../basics/daemon.md). When

> you run the `cross-seed daemon`, the app starts an HTTP server, listening on port

Reviewer: ```suggestion
you run the `cross-seed daemon` command, the app starts an HTTP server, listening on port
```

## POST `/api/announce` (experimental)

Use this endpoint to feed announces into cross-seed. For each `announce`,

> `cross-seed` will check if the given search method matches any torrents you already

Reviewer: ```suggestion
`cross-seed` will check if the provided search criteria matches any torrents you already
```

have. If found, it will run the matching algorithm to verify that the torrents
do match, and download/inject the announced torrent.

:::tip

This is a real-time alternative to scanning RSS feeds. Consider turning the RSS

> scan off (`rssCadence: undefined,`) if you set up this feature.

Reviewer: ```suggestion
scan off (`rssCadence: null,`) if you set up this feature.
```
